api_v3/lib/types/KalturaObject.php

```diff
@@ -371,23 +371,31 @@ protected function doFromObject($srcObj, KalturaDetachedResponseProfile $respons
 		$fromObjectClass = "Map_{$thisClass}_{$srcObjClass}";
 		if (!class_exists($fromObjectClass))
 		{
-			$cacheFileName = kConf::get("cache_root_path") . "/api_v3/fromObject/{$fromObjectClass}.php";
-			if (!file_exists($cacheFileName))
+			$cacheFileName = kConf::get("cache_root_path") . "api_v3/fromObject/{$fromObjectClass}.php";
```

Comment:
Why did you remove the slash before api_v3? No damage will be cause by two slashes, but you can use realpath or trim to remove extra slashes just in case.